### DISCUSSION TITLE
[Question] Wails V2: Additional Requirements: Outdated or gcc and XCode still required?

### DISCUSSION THREAD

**Discussion by MeerMusik** [wailsapp/wails#1751] [Q&A] [answered]
Hello.

I am new to Go and Wails. My Question is in the Topic Title. To be more precise:

When looking at

[https://wails.io/docs/gettingstarted/installation#platform-specific-dependencies](https://wails.io/docs/gettingstarted/installation#platform-specific-dependencies)

it says that Linux still requires GCC and macOS still requires XCode.

Is this still the case because Wails depends on / uses CGO? I am just wondering as for Windows, there is no mentioning of the Microsoft Visual C++ Compiler as an additional requirement.

My main OS is Windows. I have not tried to run Wails on Linux yet. I do not have access to a Mac.

My Infos are based on `Go 1.19.0` and `Wails v2.0.0-beta.43`.

Thank you very much in advance! :)

**Comment by leaanthony** [comment, marked as answer]
All that information is correct: Windows does not use cgo 👍

**Reply by MeerMusik** [reply]
Just to be perfectly clear:

* Linux still requires GCC
* macOS still requires XCode

Correct?

**Reply by MeerMusik** [reply]
Alright. Thank you!
